device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/https/HttpsBatchMessage.java

```diff
@@ -41,42 +43,37 @@
     /** The current number of messages in the batch. */
     private int numMsgs;
 
-    /** Constructor. Initializes the batch body as an empty JSON array. */
-    public HttpsBatchMessage()
+    public HttpsBatchMessage(List<HttpsSingleMessage> messageList) throws IotHubSizeExceededException
```

Comment:
Presumably this is not part of the public API and therefore not a breaking change?

Reply:
Correct. Users don't create this object.